1-exercises/A-this/exercise-1.js

```diff
@@ -12,15 +12,16 @@
 
 
 let person = {
-    name: "Adrian",
-    age: 32,
-    currentAddress: "London",
-    celebrateBirthday: /* todo */
+  name: "Adrian",
+  age: 32,
+  currentAddress: "London",
+  celebrateBirthday: function () {
+  return this.expectedAge;
```

Comment:
Please construct the function that will increment the age stored in the object. You need to use this keyword.